04_setup_ironic.sh

```diff
@@ -159,6 +159,29 @@ if [ ! -f "${CACHED_MACHINE_OS_IMAGE}" ]; then
   pushd $(dirname ${CACHED_MACHINE_OS_IMAGE})
   sha256sum --strict --check ${CACHED_MACHINE_OS_IMAGE}.sha256sum || ( rm -f "${CACHED_MACHINE_OS_IMAGE}" ; exit 1 )
   popd
+  # If using the live images, fetch the kernel, rootfs and initramfs images also
+  if [ "${RHCOS_LIVE_IMAGES}"  == "true" ]; then
+     CACHED_MACHINE_OS_LIVE_KERNEL="${IRONIC_DATA_DIR}/html/images/${MACHINE_OS_LIVE_KERNEL}"
+     curl -g --insecure -L -o "${CACHED_MACHINE_OS_LIVE_KERNEL}" "${LIVE_KERNEL_IMAGE_URL}"
```

Comment:
the change is ok, just wondering if we could transform this in a function since we have the exact same code repeated 3 times